packages/scripts/src/codegen/guard.ts

```diff
@@ -12,8 +13,11 @@ function run(webpack: TaskFunction, cb: TaskFunctionCallback) {
     return shell.cwd(cwd)`${cmd}`
 }
 
-export const codegenGuardWatch = series(codegen, parallel(codegenWatch, run.bind(null, extensionWatch)))
-export const codegenGuard = series(codegen, run.bind(null, extension))
+export const codegenGuardWatch: Undertaker.TaskFunction = series(
```

Comment:
`
The inferred type of 'codegenGuardWatch' cannot be named without a reference to '.pnpm/@types+undertaker@1.2.7/node_modules/@types/undertaker'. This is likely not portable. A type annotation is necessary.
`

Reply:
I'd prefer to revert this change because this package will not be type-checked.

Reply:
okay